src/services/__tests__/MatchService.unit.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { BallChaserQueueBuilder } from "../../../.jest/Builder";
+import { createRandomMatch } from "../MatchService";
+import { mocked } from "ts-jest/utils";
+import ActiveMatchRepository from "../../repositories/ActiveMatchRepository";
+import { Team } from "../../types/common";
+import { NewActiveMatchInput } from "../../repositories/ActiveMatchRepository/types";
+
+jest.mock("../../repositories/ActiveMatchRepository");
+jest.mock("../../utils/getEnvVariable");
+
+describe("Match Service tests", () => {
+  const mockBallChasers = BallChaserQueueBuilder.many(6);
+  mockBallChasers.forEach((player) => (player.team = null));
```

Comment:
Instead of doing this forEach, you can provide it as an override to the `.many` function.
```
const mockBallChasers = BallChaserQueueBuilder.many(6, { team: null });
```

Reply:
Fixed in next commit.